tests/testing_constants.py

```diff
@@ -1,6 +1,7 @@
 USER = "__DUMMY_TRANSFORMERS_USER__"
 FULL_NAME = "Dummy User"
 PASS = "__DUMMY_TRANSFORMERS_PASS__"
+TOKEN = "hf_94wBhPGp6KrrTH3KDchhKpRxZwd6dmHWLL"
```

Comment:
Is this not sensitive at all?

Reply:
not super critical, but probably cleaner to put it into a GitHub Action secret (wdyt @SBrandeis )

Reply:
Either that or use an ostensibly fake token? (like `hf_fakefakefake` or smth)

Reply:
It's still used to ensure it works. But this doesn't seem sensitive at all to me, it's exactly like the user/pass before

Reply:
it's not sensitive but it _looks_ sensitive to an outside eye. Is there any drawback to passing it through GitHub secrets (or locally through an env variable)?

Otherwise, let's add a comment below it that says "this is not sensitive because it's only on a sandboxed CI instance" or something?

Reply:
Passing it through GitHub secrets means that it will be a bit tough to run the CI locally; I'm happy to do that, but I'd prefer the comment to indicate it's not sensitive!

Reply:
SG